bookkeeper-server/src/main/java/org/apache/bookkeeper/client/LedgerHandleAdv.java

```diff
@@ -225,6 +226,21 @@ public void safeRun() {
         }
     }
 
+    @Override
+    public CompletableFuture<Long> write(long entryId, ByteBuf data) {
```

Comment:
@sijie LedgerHandleAdv did not have a ByteBuf based interface, this is a brand new method

Reply:
can we try to add one? I like to make sure all these method share same code path, rather than duplicating. otherwise it is going to be hard to maintain.